tests/expected/python/operator_pybind.cpp

```diff
@@ -0,0 +1,36 @@
+
+
+#include <pybind11/eigen.h>
+#include <pybind11/stl_bind.h>
+#include <pybind11/pybind11.h>
+#include <pybind11/operators.h>
+#include "gtsam/nonlinear/utilities.h"  // for RedirectCout.
+
+#include "gtsam/geometry/Pose3.h"
+
+#include "wrap/serialization.h"
+#include <boost/serialization/export.hpp>
+
+
+
+
+
+using namespace std;
+
+namespace py = pybind11;
+
+PYBIND11_MODULE(operator_py, m_) {
+    m_.doc() = "pybind11 wrapper of operator_py";
+
+    pybind11::module m_gtsam = m_.def_submodule("gtsam", "gtsam submodule");
+
+    py::class_<gtsam::Pose3, std::shared_ptr<gtsam::Pose3>>(m_gtsam, "Pose3")
```

Comment:
What’s happening here? Document? What if we want Rot3? Do we need to add it here as well?

Reply:
That's the generated wrapper file which is our expected value in the test. For wrapping syntax, you have to look at `fixtures/operator.i`.

For Rot3, we would have to add the `operator*` overload to the Rot3 class as well since it is effectively a class method.